Networking/Sources/Extended/Model/POSProduct.swift

```diff
@@ -20,6 +20,10 @@ public struct POSProduct: Codable, Equatable, GeneratedCopiable, GeneratedFakeab
     public let salePrice: String?
     public let onSale: Bool
 
+    public let downloadable: Bool
+
+    public let parentID: Int64
```

Comment:
thought for the future: this field (and maybe also the downloadable field) most likely isn't needed in the POS app layer, maybe we could have a separate `POSProduct` struct in the Yosemite layer public to the app layer when there are several properties internal to Yosemite.

Reply:
We already have that 😊 

The app layer doesn't consume `POSProduct`, it's for use by Networking and Yosemite only. 

Yosemite passes back `POSItem`, which has things like `POSSimpleProduct` and `POSVariableParentProduct` as associated values, depending on the enum case.

The only exception to this is when we make a fake item service for Xcode previews – the app target does import POSProduct to do that. I guess we could define that in Yosemite, but I don't think it makes too much difference really.